src/main/java/com/kakaotech/team14backend/inner/point/model/Point.java

```diff
@@ -63,7 +62,7 @@ public static Point createUserPoint(Member member, Long point){
 
   public void useUserPoint(Long usePoint){
     if(this.nowPoint - usePoint < 0){
-      throw new Exception500("보유 폭죽이 부족합니다.");
+      throw new NotEnoughPointException(MessageCode.NOT_ENOUGH_POINT);
```

Comment:
```suggestion
      throw new NotEnoughPointException();
```

예외 자기 자신이 `MessageCode.NOT_ENOUGH_POINT`를 불변 필드로 가진다면 어떨까요?

Reply:
해당 예외 클래스에서 자신의 상태를 관리할 수 있을 것 같습니다!
감사합니다!